Pipfile

```diff
@@ -34,7 +34,7 @@ tqdm = "*"
 update_checker = "*"
 semantic_version = "*"
 packaging = "*"
-cloudsplaining = ">=0.3.2"
+cloudsplaining = ">=0.4.1"
```

Comment:
thanks @njgibbon . Can you please updadte setup.py file too?

Reply:
Done that now. Also moved the test file structure around so it's not nested after some cf-lint feedback on one of the previous commits.

Reply:
I think we need too update pipfile.lock to

Reply:
Updating that file now